### PR TITLE
Fixed some sentences on the index page.

### PR DESCRIPTION
**Pull request by ZeeCoder**
I talked about these in a PR before.

### CODE REVIEW

File: pages/index.twig [hunk 1]
```diff
@@ -48,10 +48,10 @@ chmod +x /usr/local/bin/dep</code></pre>
...
                     <ul>
-                        <li><img src="/assets/icons/fast.png"><strong>Fast</strong> Deployer packed full of time-saving features and optimisations, best for performance and development.</li>
-                        <li><img src="/assets/icons/modular.png"><strong>Modular</strong> Create your deploy script from our modular blocks called recipes. You can find common recipes <a href="https://github.com/deployphp/deployer/tree/master/recipe">here</a> and 3rd party recipes <a href="https://github.com/deployphp/recipes">here</a>.</li>
-                        <li><img src="/assets/icons/cleancode.png"><strong>Clean code</strong> Deployer has very clean code and good tested. Repository is watched by many code quality tools, every pull request also checked by QA tools.</li>
-                        <li><img src="/assets/icons/rollback.png"><strong>Rollbacks</strong> If something is wrong, a simple rollback to previous release is possible.</li>
+                        <li><img src="/assets/icons/fast.png"><strong>Fast</strong> Deployer is packed with time-saving features and optimisations, best for performance and development.</li>
```
Comment (elfet) [inline pages/index.twig]: What's wrong with that?

Comment (ZeeCoder) [inline pages/index.twig]: "packed with something" is the proper term.
There's no such thing as "packed full of something", that's why I rephrased it slightly.

Comment (elfet) [inline pages/index.twig]: What you suggest?

Comment (ZeeCoder) [inline pages/index.twig]: You have two options:
 - "Deployer is full of time-saving features..."
 - "Deployer is packed with time-saving features..."

I chose the latter, it had a nicer ring to it.

Comment (elfet) [inline pages/index.twig]: Me too. Can you create PR for this?

Comment (ZeeCoder) [inline pages/index.twig]: This is the PR. :)

Comment (elfet) [inline pages/index.twig]: Sorry, look at this from main :mailbox:

**Pull request merged**